### DISCUSSION TITLE
Invited to collaborate on a repository but I get a 404 error

### DISCUSSION THREAD

**Discussion by mikeaero** [community/community#75818] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello -

I've been invited by @gregos-livecouk to collaborate on a repository but I get a 404 error when trying the link. We've gone back and forth to ensure the link hasn't expired, we've both got the right details, tried again, etc., to no avail. Help appreciated!

Thanks

**Comment by Hsin-Hung** [comment]
It is most likely an authentication issue, so check whether or not you are signed in with GitHub on the account that the invitation is for. Make sure you are logged into GitHub and once you receive the link, paste it into your browser to accept the invitation.

**Comment by yassirelkhaili** [comment]
Hey am new to github discussions but has he tried inviting you as collaborator using this method:

1. got to the repository
2. click settings
3. click collaborators
4. under manage access click add people and choose someone. that will then send this person an invite to their account.
5. on their end they have to accept this invite via their github and voila.

anyways thought I'd try to help let me know how things go.

**Comment by mikeaero** [comment]
Followed that process, checked and double-checked authentication, still no dice.


On Nov 16, 2023, at 2:09 PM, Yassir Elkhaili ***@***.***> wrote:

﻿

Hey am new to github discussions but has he tried inviting you as collaborator using this method:

  1.  got to the repository
  2.  click settings
  3.  click collaborators
  4.  under manage access click add people and choose someone. that will then send this person an invite to their account.
  5.  on their end they have to accept this invite via their github and voila.

anyways thought I'd try to help let me know how things go.

**Comment by akbaralinabiev** [comment]
Navigate to the Repository:
Go to the repository where you've been invited to collaborate.

Click on "Issues":
In the repository's menu, click on the "Issues" tab.

Click on "New Issue":
On the "Issues" page, click on the green "New Issue" button.

Choose a Template or Start a Blank Issue:
GitHub may provide templates for various types of issues. Choose a template that fits the issue you are facing, or start with a blank issue.

Provide a Clear Title:
Give your issue a clear and concise title, such as "404 Error on Collaboration Invitation Link."

Provide Detailed Information in the Body:
In the issue description, explain the problem in detail. Include information such as the steps you've taken, any error messages received, and the relevant repository and collaboration details.

Include Screenshots if Possible:
If there are specific error messages or screens, consider including screenshots. This can help others understand the issue more effectively.

Submit the Issue:
Once you've filled in all the necessary information, click on the "Submit new issue" button.

**Comment by mikeaero** [comment]
The 404 error on collaboration precludes my ability to achieve step 1 (navigate to the repository.)

From: Akbarali Nabiev ***@***.***>
Date: Thursday, November 16, 2023 at 4:23 PM
To: community/community ***@***.***>
Cc: Michael R Moore ***@***.***>, Author ***@***.***>
Subject: Re: [community/community] Invited to collaborate on a repository but I get a 404 error (Discussion #75818)

Navigate to the Repository:
Go to the repository where you've been invited to collaborate.

Click on "Issues":
In the repository's menu, click on the "Issues" tab.

Click on "New Issue":
On the "Issues" page, click on the green "New Issue" button.

Choose a Template or Start a Blank Issue:
GitHub may provide templates for various types of issues. Choose a template that fits the issue you are facing, or start with a blank issue.

Provide a Clear Title:
Give your issue a clear and concise title, such as "404 Error on Collaboration Invitation Link."

Provide Detailed Information in the Body:
In the issue description, explain the problem in detail. Include information such as the steps you've taken, any error messages received, and the relevant repository and collaboration details.

Include Screenshots if Possible:
If there are specific error messages or screens, consider including screenshots. This can help others understand the issue more effectively.

Submit the Issue:
Once you've filled in all the necessary information, click on the "Submit new issue" button.
